Modules/_datetimemodule.c

```diff
@@ -3946,6 +3960,9 @@ static PyMethodDef date_methods[] = {
      PyDoc_STR("Return the day of the week represented by the date.\n"
                "Monday == 0 ... Sunday == 6")},
 
+     {"timestamp", (PyCFunction)date_timestamp, METH_NOARGS,
```

Comment:
```suggestion
     {"timestamp", date_timestamp, METH_NOARGS,
```

This cast is not necessary